.npmrc

```diff
@@ -1 +1 @@
-@sugarlabs:registry=https://npm.pkg.github.com
+//npm.pkg.github.com/:_authToken=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
```

Comment:
I guess now you need to change your PAT for security reasons 😛. This doesn't go here. Please revisit the setup guide.